package cobo.chat.data.exception.chatGPT

class InvalidChatGPTResException(message: String): Exception(message) {
}